src/core/map/hooks/useMapClickHandler.ts

```
@@ -0,0 +1,56 @@
+import { useEffect } from 'react';
+import type { Map } from 'maplibre-gl';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)